src/App.vue

```diff
@@ -59,7 +59,7 @@
 				min="1"
 				max="128"
 				maxlength="2"
-				label="Mask"
+				:label="t('bruteforcesettings','Mask')"
```

Comment:
Should add a translation hint for translators that this is about IP address masks and not face masks?

Reply:
https://github.com/nextcloud/bruteforcesettings/pull/554

Reply:
Ref https://docs.nextcloud.com/server/latest/developer_manual/basics/front-end/l10n.html#provide-context-hints-for-translators